backend/src/main/java/ca/gov/dtsstn/cdcp/api/web/util/Functions.java

```diff
@@ -0,0 +1,26 @@
+package ca.gov.dtsstn.cdcp.api.web.util;
+
+import org.springframework.http.HttpStatus;
+import org.springframework.stereotype.Component;
+import org.springframework.web.server.ResponseStatusException;
+
+import com.fasterxml.jackson.core.JsonProcessingException;
+import com.fasterxml.jackson.databind.JsonNode;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import com.github.fge.jsonpatch.JsonPatch;
+import com.github.fge.jsonpatch.JsonPatchException;
+
+
+@Component
+public class Functions {
+
+	public <T> T applyJsonPatch(JsonPatch patch, T targetModel, Class<T> targetModelClass) {
+		try {
+			final ObjectMapper objectMapper = new ObjectMapper();
```

Comment:
I feel like this should be a class field, since instantiating it is *very* expensive.

``` java
private final ObjectMapper objectMapper = new ObjectMapper()
    .disable(DeserializationFeature.FAIL_ON_UNKNOWN_PROPERTIES)
    .findAndRegisterModules();
```

Reply:
added

Reply:
It should be a class field, not a method-scoped variable.

Note that this whole class should be removed when #1710 is merged.

Reply:
removed after #1710 was merged.